hls-plugin-api/hls-plugin-api.cabal

```diff
@@ -44,6 +44,7 @@ library
     , ghc-boot-th
     , ghcide                >=0.4
     , haskell-lsp           ^>=0.22
+    , hashable
```

Comment:
Why is this required?

Reply:
The client settings are stored in ghcide as `Hashable` so we need it to extract the value here:
https://github.com/haskell/haskell-language-server/pull/166/files#diff-ee87f10cf4f1e80aa153e4cadeb50997b93dba31aa8d685fce255ca7a38c7191R599